citesphere/src/main/java/edu/asu/diging/citesphere/web/user/authorities/AuthorityEntryController.java

```diff
@@ -63,9 +63,9 @@
         return new ResponseEntity<AuthoritySearchResult>(authorityResult, HttpStatus.OK);
     }
     
-    @RequestMapping("/auth/authority/{zoteroGroupId}/find/authorities/{source}")
+    @RequestMapping("/auth/authority/find/authorities/{source}")
```

Comment:
instead of changing the url and risking missing a spot where it wouldn't work anymore, you could just add both paths to the request mapping (so that the method respond to both urls) and leave the existing code untouched